domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -694,6 +700,7 @@ public void endTx(AsyncResult<SQLConnection> trans, Handler<AsyncResult<Void>> d
       if (sqlConnection.conn != null) {
         sqlConnection.conn.close();
       }
+      cancelConnectionTimeoutTimer(sqlConnection);
```

Comment:
Can you make closeAndHandleResult and cancelConnectionTimeoutTimer static?

Reply:
cancelConnectionTimeoutTimer uses _vertx_ from the PostgresCient instance

Reply:
Yes, ok, I havn't seen that.